images/dcos/bootstrap.sh

```diff
@@ -163,6 +154,32 @@ EOF
   prepare_service_depends etcd-server ${etcd_server_list}/v2/stats/store getsSuccess ${deps}
 }
 
+prepare_etcd_proxy() {
+  prepare_service ${monitor_dir} ${service_dir} etcd-server ${ETCD_SERVER_RESPAWN_DELAY:-1} << EOF
+#!/bin/sh
+#TODO(jdef) don't run this as root
+#TODO(jdef) would be super-cool to have socket-activation here so that clients can connect before etcd is really ready
+exec 2>&1
```

Comment:
using env variables in execlineb sucks. if we get rid of PATH then we could
migrate to execlineb

On Wed, Oct 14, 2015 at 11:15 AM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In images/dcos/bootstrap.sh
> <https://github.com/mesosphere/kubernetes-mesos/pull/556#discussion_r42007024>
> :
>
> > @@ -163,6 +154,32 @@ EOF
> >    prepare_service_depends etcd-server ${etcd_server_list}/v2/stats/store getsSuccess ${deps}
> >  }
> >
> > +prepare_etcd_proxy() {
> > +  prepare_service ${monitor_dir} ${service_dir} etcd-server ${ETCD_SERVER_RESPAWN_DELAY:-1} << EOF
> > +#!/bin/sh
> > +#TODO(jdef) don't run this as root
> > +#TODO(jdef) would be super-cool to have socket-activation here so that clients can connect before etcd is really ready
> > +exec 2>&1
>
> that's the point. Why are all other scripts using execlineb and this is
> not?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/kubernetes-mesos/pull/556/files#r42007024>.
>